CorsixTH/Lua/hospital.lua

```diff
@@ -32,18 +32,25 @@ function Hospital:Hospital(world, avail_rooms, name)
   local level_config = world.map.level_config
   local level = world.map.level_number
   local balance = 40000
-  local interest_rate = 0.01
-  if level_config then
-    if level_config.towns and level_config.towns[level] then
-      balance = level_config.towns[level].StartCash
-      interest_rate = level_config.towns[level].InterestRate / 10000
-    elseif level_config.town then
-      balance = level_config.town.StartCash
-      interest_rate = level_config.town.InterestRate / 10000
-    end
+  local reputation_min = 0
+  local reputation_max = 1000
+  local reputation = 500
+  local interest_rate, overdraft_differential
+
+  assert(level_config, "Cannot load level configuration.")
```

Comment:
Should we assert this? In map.lua if we fail to load a level config the base_config takes its place instead.